ArkhamOverlay/Data/CardGroup.cs

```diff
@@ -80,17 +74,25 @@ public class CardGroup : ViewModel, ICardGroup, INotifyPropertyChanged {
             get => _buttonImageAsBytes;
             set {
                 _buttonImageAsBytes = value;
-                _eventBus.PublishCardGroupChanged(Id, Name, value != null, Name);
+                PublishCardGroupChanged();
             }
         }
 
+        /// <summary>
+        /// Notify listeners that the card group has changed
+        /// </summary>
+        public void PublishCardGroupChanged() {
+            var zoneNames = (from zone in _cardZones select zone.Name).ToList();
```

Comment:
Since you're using linq, it's probably better to use this:
```suggestion
            var zoneNames = _cardZones.Select(x => x.Name).ToList();
```
I'd also be a fan of using `IEnumerable` instead of `IList` to save us those` ToList` calls

Reply:
I changed it the consistent syntax. I am passing back a list because ultimately this gets serialized to json, and I just want to be explicit about the timing of things, especially on the other side where it is deserialized. I don't know how newstonsoft recreates IEnumerables, but it could create something very ineffecient since I am storing the ressults on the other end and using them frequently. In general, converting to a list is a good practice, I think, because you know it won't have to resulve the select logic multiple times.

Reply:
Gotcha. Keeping the `ToList` methods seems good then.